lib/srv/desktop/tdp/proto.go

```diff
@@ -1631,6 +1636,44 @@ func decodeSharedDirectoryTruncateResponse(in io.Reader) (SharedDirectoryTruncat
 	return res, err
 }
 
+// LatencyStats is used to report the latency of the connection(s) to the web UI.
```

Comment:
Soon we will have desktop access in Connect (I assume we will add the latency monitor there at some point), so I’d avoid referring specifically to the Web UI and instead use something like 'client' instead.

Reply:
Yep, changed